### ISSUE TITLE
Add persistent entity api

### ISSUE THREAD

**Issue by Fusezion** [SkriptLang/Skript#7154]
### Suggestion

Add the persistent entity api

### Why?

Not all entities need to be saved between server restarts and when you're constantly needing to try and prevent the edge cases where they are it's annoying.

As such having syntax like below will be a huge help for servers
```applescript
%entities% (is|are) persistent
%entities% (isn't|is not|aren't|are not) persistent
make %entities% [:non[-| ]]persistent
```

### Other

[Entity#setPersistent(boolean)](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Entity.html#setPersistent(boolean)) and [Entity#isPersistent()](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Entity.html#isPersistent())

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by Asleeepp** [comment]
Theres a pr for this (https://github.com/SkriptLang/Skript/pull/5296) but its waiting on another pr to merge to come out of draft

**Comment by Fusezion** [comment]
not a fan of the requirement of #5219 if a member of the lang is willing to compromise and accept a pr without that requirement I'd be happy
